seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oracle/OracleCatalog.java

```diff
@@ -165,62 +147,30 @@ protected String getSelectColumnsSql(TablePath tablePath) {
     @Override
     protected Column buildColumn(ResultSet resultSet) throws SQLException {
```

Comment:
| COLUMN\_NAME | TYPE\_NAME | FULL\_TYPE\_NAME | COLUMN\_LENGTH | COLUMN\_PRECISION | COLUMN\_SCALE | COLUMN\_COMMENT | DEFAULT\_VALUE | IS\_NULLABLE |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| F1 | NUMBER | NUMBER | 22 | null | 0 | null | null | NO |
| F2 | NUMBER | NUMBER | 22 | null | null | null | null | YES |
| F3 | NUMBER | NUMBER\(8, 0\) | 22 | 8 | 0 | null | null | YES |
| F4 | NUMBER | NUMBER\(18, 0\) | 22 | 18 | 0 | null | null | YES |
| F5 | NUMBER | NUMBER\(38, 0\) | 22 | 38 | 0 | null | null | YES |
| F6 | NUMBER | NUMBER\(10, 2\) | 22 | 10 | 2 | null | null | YES |
| F7 | FLOAT | FLOAT | 22 | 126 | null | null | null | YES |
| F8 | BINARY\_FLOAT | BINARY\_FLOAT | 4 | null | null | null | null | YES |
| F9 | FLOAT | FLOAT | 22 | 63 | null | null | null | YES |
| F10 | BINARY\_DOUBLE | BINARY\_DOUBLE | 8 | null | null | null | null | YES |
| F11 | CHAR | CHAR\(1\) | 1 | null | null | null | null | YES |
| F12 | CHAR | CHAR\(10\) | 10 | null | null | null | null | YES |
| F13 | NCHAR | NCHAR\(2\) | 2 | null | null | null | null | YES |
| F14 | NCHAR | NCHAR\(20\) | 20 | null | null | null | null | YES |
| F16 | VARCHAR2 | VARCHAR2\(10\) | 10 | null | null | null | null | YES |
| F18 | NVARCHAR2 | NVARCHAR2\(20\) | 20 | null | null | null | null | YES |
| F19 | XMLTYPE | XMLTYPE | 2000 | null | null | null | null | YES |
| F20 | LONG | LONG | 0 | null | null | null | null | YES |
| F21 | CLOB | CLOB | 4000 | null | null | null | null | YES |
| F22 | NCLOB | NCLOB | 4000 | null | null | null | null | YES |
| F23 | BLOB | BLOB | 4000 | null | null | null | null | YES |
| F25 | RAW | RAW\(10\) | 10 | null | null | null | null | YES |
| F27 | DATE | DATE | 7 | null | null | null | null | YES |
| F28 | TIMESTAMP | TIMESTAMP\(6\) | 11 | null | 6 | null | null | YES |
| F29 | TIMESTAMP | TIMESTAMP\(6\) | 11 | null | 6 | null | null | YES |
| F30 | TIMESTAMP | TIMESTAMP\(6\) WITH TIME ZONE | 13 | null | 6 | null | null | YES |
| F31 | TIMESTAMP | TIMESTAMP\(6\) WITH TIME ZONE | 13 | null | 6 | null | null | YES |
| F32 | TIMESTAMP | TIMESTAMP\(6\) WITH LOCAL TIME ZONE | 11 | null | 6 | null | null | YES |
| F33 | TIMESTAMP | TIMESTAMP\(6\) WITH LOCAL TIME ZONE | 11 | null | 6 | null | null | YES |